universal-application-tool-0.0.1/test/auth/ProfileMergeTest.java

```diff
@@ -27,8 +27,8 @@ public void setupFactory() {
     UserRepository repository = instanceOf(UserRepository.class);
     idcsProfileAdapter =
         new IdcsProfileAdapter(
-            null,
-            null,
+            /* configuration */ null,
```

Comment:
Please add a = after each name.    We'll be warned if the name is wrong (EG we're not setting what we think we are.)

I just found out that this is enforced by the complier, I thought it was a special tool internally.